src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -271,8 +324,8 @@ function genFragment(
   // BR tags
   if (nodeName === 'br') {
     if (
-        lastLastBlock === 'br' &&
-        (!inBlock || getBlockTypeForTag(inBlock, lastList) === 'unstyled')
+      lastLastBlock === 'br' && (!inBlock ||
+      getBlockTypeForTag(inBlock, lastList, draftBlockRenderMap) === 'unstyled')
```

Comment:
I don't mind it, we should probably add this as a rule to eslintrc so that the formatting can be picked up on the linting